WickedEngine/CMakeLists.txt

```diff
@@ -103,6 +103,34 @@ set(WICKEDENGINE_STATIC_LIBRARIES
 	Utility
 )
 
+if (USE_AVX512)
```

Comment:
I'd like to have JPH defines here hard coded instead of defines depending on other defines.

Reply:
there's little need, as the JPH will already be defined when compiling Jolt.

It would only be needed if wiMath sets _XM_ depending on JPH_ flags, which as mentioned above is weird; alternatively, these can define the _XM_ flags, this way, nothing is needed in wiMath